lib/streamlit/type_util.py

```diff
@@ -287,6 +292,16 @@ def is_pydeck(obj: object) -> TypeGuard[Deck]:
     return is_type(obj, "pydeck.bindings.deck.Deck")
 
 
+def is_pydantic_model(obj) -> bool:
+    """True if input looks like a Pydantic model instance."""
+
+    if isinstance(obj, type):
+        # Should be an instance, not a class.
```

Comment:
based on the comment I would expect a `if not isinstance` check 🤔

Reply:
oh, yeah. The comment is a bit misleading: `isinstance(obj, type)` -> means that it is a `class` and we need to return False

Reply:
I will update the comment

Reply:
Updated 👍